platform/nodejs/package.json

```diff
@@ -30,6 +30,7 @@
     "npm": ">=6.14.4"
   },
   "devDependencies": {
-    "eslint": "^7.32.0"
+    "eslint": "^7.32.0",
+    "mocha": "^9.0.3"
```

Comment:
@gorhill I'm introducing the Mocha framework here.

I have left the `test.js` file as it is for now, but in the final version of this patch, if you like the idea, we can remove code from that file.

Please see other inline comments.